<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>U2::ExportMSA2MSADialog</class>
 <widget class="QDialog" name="U2::ExportMSA2MSADialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>278</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Export Nucleic Alignment to Amino Translation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="fileLabel">
       <property name="text">
        <string>Export to file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fileNameEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="fileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="formatLabel">
       <property name="text">
        <string>File format to use</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="formatCombo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="translationLabel">
       <property name="text">
        <string>Amino translation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="translationCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="addDocumentButton">
     <property name="text">
      <string>Add document to the project</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="rangeGroupBox">
     <property name="title">
      <string>Export range</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="wholeRangeButton">
        <property name="text">
         <string>Whole alignment</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="selectedRangeButton">
        <property name="text">
         <string>Selected rows</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>U2::ExportMSA2MSADialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
